tools/docker-images/clp-package/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:jammy AS base
+
+WORKDIR /root
+
+COPY ./tools/docker-images/clp-package/setup-scripts ./setup-scripts
+RUN ./setup-scripts/install-prebuilt-packages.sh \
+    && rm -rf ./setup-scripts/
+
+# Remove cached files
+RUN apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+ENV CLP_HOME="/opt/clp"
+ENV PATH="${CLP_HOME}/bin:${PATH}"
+ENV PATH="${CLP_HOME}/sbin:${PATH}"
+ENV PYTHONPATH="${CLP_HOME}/lib/python3/site-packages"
+
+USER 1000:1000
+
+COPY ./build/clp-package /opt/clp
+
+# Flatten the image
+FROM scratch
+COPY --from=base / /
```

Comment:
While currently the `USER` and `ENV` commands are unused, their effects would be lost after flattening the image. i.e., Those settings should be done after the image is flattened.

@coderabbitai create an issue to track this